DEV-9/SymbolReplacementInStrings/EntryPoint.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace SymbolReplacementInStrings
+{
+    /// <summary>
+    /// Entrypoint to program.
+    /// </summary>
+    class EntryPoint
+    {
+        /// <summary>
+        /// This method calls one method, that reads two strings from the file
+        /// and another method, that randomly replaces symbols in strings.
+        /// </summary>
+        /// <param name="args">Argument contains the path to the file with strings.</param>
+        static void Main(string[] args)
+        {
+            const int linesCount = 2;
+            const string NOTENOUGHLINESINTHEFILE = "There is't not enough lines. Please, add two lines in the file.";
+
+            try
+            {
+                List<string> lines = new List<string>();
```

Comment:
????? why you allocate memory for lines here? TextReader returns allocated list?